decision_service/dsclientlib/ds_api_status.cc

```diff
@@ -0,0 +1,41 @@
+
+#include "ds_api_status.h"
+
+
+namespace decision_service {
+
+	int api_status::get_error_code() const {
+		return _error_code;
```

Comment:
In general this makes sense.  Since api_status is part of our public interface, does it make sense to keep all implementation in cc files, especially since we might be shipping binaries?

Reply:
I agree.